src/class/video/video_device.c

```diff
@@ -408,10 +480,10 @@ static bool _negotiate_streaming_parameters(videod_streaming_interface_t const *
         break;
       default: return false;
     }
-    param->bFrameIndex = (uint8_t) frmnum;
+    param->bFrameIndex = (uint8_t)frmnum;
     /* Set the parameters determined by the frame */
     tusb_desc_cs_video_frm_uncompressed_t const *frm = _find_desc_frame(tu_desc_next(fmt), end, frmnum);
```

Comment:
line 470, 485, 494, 495 should all have the pointer type changed for consistency as well as functionality (in some cases)

`tusb_desc_cs_video_frm_uncompressed_t` --> `tusb_desc_cs_video_frm_t`

`tusb_desc_cs_video_fmt_uncompressed_t` --> `tusb_desc_cs_video_fmt_t`